### PR TITLE
Simplify new hub template

### PR DESCRIPTION
**Pull request by choldgraf**
This simplifies our "new hub" template slightly, so that the final issue takes up less vertical space, and so that we chunk together options in `textarea` blocks rather than having a dedicated field for each. Its goal is to make the new hub issue a bit less confusing, and make it easier to find the right information.

Here's a link to the rendered version: https://github.com/2i2c-org/infrastructure/blob/choldgraf-patch-1/.github/ISSUE_TEMPLATE/2_new-hub.yml

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_new-hub.yml [hunk 1]
```diff
@@ -115,56 +67,55 @@ body:
       options:
         - GitHub Authentication (e.g., @mygithubhandle)
         - Google OAuth (e.g., myemailaddress@2i2c.org)
+        - CILogon (yy
```
Comment (sgibson91) [inline .github/ISSUE_TEMPLATE/2_new-hub.yml:70]:
```suggestion
        - CILogon (yy
```

Was this an unfinished thought?

File: .github/ISSUE_TEMPLATE/2_new-hub.yml [hunk 2]
```diff
@@ -115,56 +67,55 @@ body:
...
         - Username / Password via [auth0](https://auth0.com/docs/connections/database)
```
Comment (sgibson91) [inline .github/ISSUE_TEMPLATE/2_new-hub.yml:71]:
```suggestion
        - Username / Password via auth0: https://auth0.com/docs/connections/database
```

The link doesn't render properly in the drop down

Screenshot 2022-02-10 at 10 04 08

**Review by sgibson91 — APPROVED** [review]
Couple of fixes to make, but I like it!

File: .github/ISSUE_TEMPLATE/2_new-hub.yml [hunk 3]
```diff
@@ -115,56 +67,55 @@ body:
...
-    validations:
-      required: false
-
-  - type: input
-    id: hub-logo-url
-    attributes:
-      label: Hub logo URL
-      description: |
-        A URL that the hub's logo will point to when users click on it on the landing page (optional)
+        The hub landing page will have its own logo and a link attached to it. This generally points to a community or institutional website.
```
Comment (GeorgianaElena) [inline .github/ISSUE_TEMPLATE/2_new-hub.yml:81]: Maybe rephrase this to make it more clear what we mean?
```suggestion
        The hub landing page can have a specific logo and a link attached to it. This generally points to a community or institutional website.
```

**Review by GeorgianaElena — APPROVED** [review]
This is great!  :rocket:

File: .github/ISSUE_TEMPLATE/2_new-hub.yml [hunk 4]
```diff
@@ -46,65 +46,17 @@ body:
...
-      description: |
-        If you have restrictions about which cloud provider to use, please indicate here. **Only choose a cloud provider if you have a strong opinion**. You may wish to specify a specific provider if you have a dataset that exists in one of the cloud providers already, or a billing account that you wish to use with a pre-existing cloud provider. If you have no preference, we'll use Google Cloud Platform.
-      options:
-        - No preference (default)
-        - Google Cloud Platform
-        - Amazon Web Services
-        - Microsoft Azure
+        - **Required start date**: <!-- the hub MUST by running as-needed by this date -->
+        - **Target start date**: <!-- choose a target date ~1 week earlier than the required date -->
+        - **Any important dates for usage**: For example - exams, periods of heavy usage, etc.
```
Comment (choldgraf) [inline .github/ISSUE_TEMPLATE/2_new-hub.yml:52]:
```suggestion
        - **Any important dates for usage**: <!-- For example - exams, periods of heavy usage, etc. -->
```

File: .github/ISSUE_TEMPLATE/2_new-hub.yml (hunk 1 above)
Comment (choldgraf) [reply .github/ISSUE_TEMPLATE/2_new-hub.yml:70]: good catch! will make an edit

**Comment by choldgraf** [comment]
Thanks for these comments and improvements! I've fixed them all I believe...let me know if there's other stuff you'd like to see in this PR!

**Comment by choldgraf** [comment]
good call @sgibson91 - I made that fix and will merge this in!
